terraform/network.tf

```diff
@@ -0,0 +1,60 @@
+data "aws_availability_zones" "available" {}
```

Comment:
What do you think about using existing module for creating VPC and all network stuff https://registry.terraform.io/modules/terraform-aws-modules/vpc/aws/1.37.0?

Reply:
You're right. Using some community modules makes sense.